packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -55,6 +55,20 @@ abstract class Scrollable extends StatefulComponent {
   /// The axis along which this widget should scroll.
   final Axis scrollDirection;
 
+  /// Whether to place zeroth child at the start of the container or
+  /// the end of it, when the scrollable has not been scrolled and has
+  /// no initial scroll offset.
+  ///
+  /// For example, if the [scrollDirection] is [Axis.vertical], then
+  /// [ViewportAnchor.start] means that the top of the zeroth item
+  /// should be aligned with the top of the scrollable, and
+  /// [ViewportAnchor.end] means the bottom of the zeroth item should
+  /// be aligned with the bottom of the scrollable.
```

Comment:
This isn't accurate.  If there are 5 items, the bottom of the 5th item will be aligned with the bottom of the scrollable.